### PR TITLE
[NONPR-2764][DC] upgrade play version

### PR DESCRIPTION
**Pull request by DanielConnelly**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -30,7 +30,7 @@ def test(scope: String) = Seq(
   "org.scalatestplus" %% "mockito-1-10" % "3.1.0.0" % Test,
   "org.jsoup" % "jsoup" % "1.13.1" % scope,
   "com.vladsch.flexmark" % "flexmark-all" % "0.35.10" % scope,
-  "com.typesafe.akka" %% "akka-testkit" % "2.6.19" % scope,
+  "com.typesafe.akka" %% "akka-testkit" % "2.6.20" % scope,
```
Comment (greg-gajda) [inline build.sbt:33]: is this one still apache licensed?

**Review by greg-gajda — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (DanielConnelly) [reply build.sbt:33]: Good question! let me check.

Comment (DanielConnelly) [reply build.sbt:33]: From here: https://github.com/akka/akka/releases 

It appears that 2.7.x is when the licence changes. 

So I think we are safe for now. ideally we'd look to remove this entirely? I can't remember how the conversation went.

Comment (greg-gajda) [reply build.sbt:33]: I don't think there was any conclusion about akka future on nrs project

**Pull request merged**